AssetsPickerViewController/Classes/Picker/AssetsPickerColors.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AssetsPickerColors.swift
+//  
+//
+//  Created by Paolo Rau on 10/12/22.
+//
+import UIKit
+
+/// Class to hold color values for Picker elements.
+public struct AssetsPickerColors {
+  var defaultCheckmark: UIColor
```

Comment:
is the default checkmark used?

Reply:
Good catch, I forgot to move `SSCheckMark.swift`